src/ONYXKEYS.js

```diff
@@ -222,6 +222,9 @@ export default {
     // Whether the auth token is valid
     IS_TOKEN_VALID: 'isTokenValid',
 
+    // The color theme setting defined by the user in preferences
+    COLOR_THEME: 'colorTheme',
```

Comment:
This also may need to be renamed. I'm thinking `NVP_PREFERRED_THEME` or `PREFERRED_THEME` to match the naming convention of the key in the user's NVPS